apps/cms.c

```diff
@@ -339,6 +339,7 @@ int cms_main(int argc, char **argv)
     if (encerts == NULL || vpm == NULL)
         goto end;
 
+    opt_set_unknown_name("cipher");
```

Comment:
Why not make the default unknown name be `cipher` and then this improvement is much less invasive. Supporting multiple "-*" options isn't necessary.